molgenis/molgenis-emx2
# feat: docker compose add ssr catalogue

What are the main changes you did:
- The docker compose file did not include the new ssr catalogue
- Added the services nginx as reverse-proxy and molgenis-ssr-catalogue
- moved the environment to a .env file (see .env-sample) 
- Added nginx default config template for ssr 

how to test:
- Create a .env file (use .env-sample)
- Make sure psql_data folder is not present (if you used docker compose previously)
- docker compose up (or docker compose up -d for detached)
- open a browser and navigate to localhost
- you schould be redirected to: localhost/apps/central/#/
- Log in as admin: localhost/apps/central/#/admin
- Navigate back to: http://localhost/apps/central/#/
- Create a new database by clicking on +
- name: catalogue, template: DATA_CATALOGUE, load example data: true; press Create database (get a cup of coffee)
- Spinner might get stuck so navigate in new tab to localhost
- If you see the database catalogue then proceed to next step
- Click on sign out
- Navigate to catalogue: http://localhost/catalogue/catalogue/#/
- In the top menu bar click SSR Catalogue
- In the top menu click 'ALL VARIABLES'
- Check if the theme is present and you get the list of variables
- close docker compose by ctrl-c or docker compose stop

todo:
- [na] added/updated tests
- [na] added/updated testplan to include a test for this fix, including ref to bug using # notation
